student_organization/__manifest__.py

```diff
@@ -0,0 +1,78 @@
+{
+    'name': 'Student Organization',
+    'version': '1.0',
+    'category': 'Hospitality',
+    'author': 'Odoo S.A.',
+    'depends': [
+        'appointment',
+        'base_industry_data',
+        'documents_sign',
+        'hr_expense',
+        'knowledge',
+        'partnership',
+        'pos_enterprise',
+        'project_enterprise',
+        'purchase',
+        'sale_planning',
```

Comment:
I see in knowledge that planning is in the "go beyond" section, so I doubt this is needed
```suggestion
```